packages/react-aria-components/src/Table.tsx

```diff
@@ -1084,13 +1099,17 @@ function TableRow<T>({item}: {item: GridNode<T>}) {
         ref={ref}
         data-disabled={states.isDisabled || undefined}
         data-selected={states.isSelected || undefined}
+        // TODO: should data-hovered reflect the previous behavior of only being applied to interactive elements
+        // so as to not break people's current implementations? Feels kinda weird though naming wise though...
```

Comment:
The changes in this PR mean that people's current implementations that rely on the current `data-hovered` behavior of NOT applying for non-interactive elements will break. Perhaps `data-hovered` should stay with the old behavior? But then what would be call the new data-attribute (it would need to be hover specific)...

Reply:
yeah it's a breaking change. you would have to change it to use `[data-interactive][data-hovered]` if you want the current behavior.